src/main/webapp/download/index.jsp

```diff
@@ -313,6 +313,7 @@
     <li><a href="https://github.com/admc/wd">Javascript</a> by Adam Christian</li>
     <li><a href="https://github.com/jlipps/yiewd">Javascript</a> by Jonathan Lipps</li>
     <li><a href="http://webdriver.io">Javascript</a> by Camilo Tapia, Vincent Voyer and Christian Bromann</li>
+    <li><a href="https://theintern.github.io/leadfoot/">Leadfoot</a> by SitePen</li>
```

Comment:
Thanks for the catch @lukeis , clearly I wasn't awake yet with my very early morning PR. ;)